resources/serverless/values.yaml

```diff
@@ -156,7 +156,7 @@ containers:
       functionBuildExecutorImage:
         value: gcr.io/kaniko-project/executor:v1.3.0
       functionBuildRepoFetcherImage:
-        value: eu.gcr.io/kyma-project/function-build-init:PR-10558
+        value: eu.gcr.io/kyma-project/function-build-init:f27d4182
```

Comment:
We could use also `8a8658b2`, because we build all images together.

Reply:
But there were no changes to that image in that PR, so if I wanted to see the changes based on that commit id then I'd find nothing